api/blueprints/drbCitation.py

```diff
@@ -59,6 +60,777 @@ def citationFetch(uuid):
 
 def mlaGenerator(citationWork):
     if citationWork:
-        return f'A FORMATTED CITATION {citationWork.title}'
+        if citationWork.title == "The Bible":
+            return mlaBibleGenerator(citationWork)
+
+        elif citationWork.editions[0].measurements:
+            if citationWork.editions[0].measurements[0] == {"type": "government_document", "value": "1"}:
+                return mlaGovGenerator(citationWork)
+
+        elif len(citationWork.authors) == 0:
+            return mlaNoAuthorGenerator(citationWork)
+
+        elif len(citationWork.authors) == 1:
+            return mlaOneAuthorGenerator(citationWork)
+
+        elif len(citationWork.authors) == 2:
+            return mlaTwoAuthorsGenerator(citationWork)
+
+        elif len(citationWork.authors) == 3:
+            return mlaThreeAuthorsGenerator(citationWork)
```

Comment:
I think we can de-duplicate a lot of code in these methods! The logic as implemented looks good, but I think we can abstract it away into a smaller class so that you don't have to repeat yourself so much. This is generally a good practice because it makes maintenance easier and helps make the code more readable.